nativescript-angular/testing/setup.js

```diff
@@ -0,0 +1,19 @@
+require("reflect-metadata");
+require("nativescript-angular/zone-js/dist/zone-nativescript");
+require("zone.js/dist/long-stack-trace-zone");
+require("zone.js/dist/proxy"); // since zone.js 0.6.15
+require("zone.js/dist/sync-test");
+if (global.jasmine) {
+    require("zone.js/dist/jasmine-patch"); // put here since zone.js 0.6.14
+} 
+require("zone.js/dist/async-test");
+require("zone.js/dist/fake-async-test");
+var testing_1 = require("@angular/platform-browser-dynamic/testing");
```

Comment:
i started with TypeScript, but had some issues with module dependencies, but will check it again